microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -282,8 +283,24 @@ Whether to include event initiators in the Timeline.
 <!-- todo: need text -->
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 113.
-*  This checkbox is present in Microsoft Edge Stable 111.
+*  This checkbox is present in Microsoft Edge Canary 114.
+*  This checkbox is present in Microsoft Edge Stable 112.
+
+
+<!-- ====================================================================== -->
+## Enable JavaScript Profiler temporarily
+
+_new as of Microsoft Edge 114_
+
+Temporarily enables the JavaScript Profiler.
+<!-- todo: need text -->
```

Comment:
@captainbrosset @zoherghadyali need text about new checkbox "Enable JavaScript Profiler temporarily"

https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript-profiler/javascript-profiler-tool - The **JavaScript Profiler** tool has been replaced by the **Performance** tool and the **Memory** tool.

Reply:
kept as-is